config/betterquesting/DefaultQuests.json

```diff
@@ -6956,105 +7308,1207 @@
           "taskID:8": "bq_standard:retrieval"
         }
       }
-    }
-  },
-  "questLines:9": {
-    "0:10": {
-      "lineID:3": 0,
-      "order:3": 0,
+    },
+    "106:10": {
+      "preRequisites:11": [
+        20
+      ],
       "properties:10": {
         "betterquesting:10": {
-          "bg_image:8": "",
-          "bg_size:3": 256,
-          "desc:8": "No Description",
+          "autoclaim:1": 0,
+          "desc:8": "Item Pipes are a simple yet very effective way to move items around §dinstantly§r, but with throughput limitations.\nHigher tier pipes provide ridiculously large throughput, a rare sight in modded Minecraft!\n\nItem Pipes §9do not pull§r, but they will push any item inserted into them to other connected inventories, prioritizing §9the closest§r first.\nThe priority stat is less important, but it may come handy for medium size pipe networks.\n\n\nRight now, you can use them in combination with §6Hoppers§r, but you will also be able to use them with Conveyor Module, Output Bus, and machine Auto-Output all in LV age.\n\nYou can use your Wrench to disconnect a side of a pipe.\n\n\n§bLore:§r Oooooh YEAH! Item Pipes were in GregTech 5, and for whatever reason they were not kept for GTCE, so you had to use other mods to transport items. Who does that!? Jokes aside, I hope you love Item Pipes being back in GregTech!",
```

Comment:
```suggestion
          "desc:8": "Item Pipes are a simple yet very effective way to move items around §dinstantly§r, but with throughput limitations.\nHigher tier pipes provide ridiculously large throughput, a rare sight in modded Minecraft!\n\nItem Pipes §9do not pull§r, but they will push any item inserted into them to other connected inventories, prioritizing §9the closest§r first.\nThe priority stat is less important, but it may come handy for medium size pipe networks.\n\n\nRight now, you can use them in combination with §6Hoppers§r, but you will also be able to use them with Conveyor Modules, Output Buses, and machine Auto-Outputs all in the LV age.\n\nYou can use your Wrench to disconnect a side of a pipe.\n\n\n§bLore:§r Oooooh YEAH! Item Pipes were in GregTech 5, and for whatever reason they were not kept for GTCE, so you had to use other mods to transport items. Who does that!? Jokes aside, I hope you love Item Pipes being back in GregTech!",
```

Also do we want to explain the priority mechanic? Basically items will always prefer the path of the lowest total priority number. Nothing else really needs to be said about it other than that it only applies when not doing round-robin.